app/api/error_handler.rb

```diff
@@ -0,0 +1,10 @@
+class ErrorHandler < Grape::Middleware::Base
+  def call!(env)
+    @env = env
+    begin
+      @app.call(@env)
+    rescue StandardError => e
```

Comment:
I'd not catch all errors thru StandardError, it's too general. We'll lose visibility about errors that happen in our application.

So, I'd recommend catching exact errors: ActiveRecord::RecordNotFound, ActionController::ParameterMissing, ActiveModel::ValidationError